changelogs/current.yaml

```diff
@@ -19,6 +19,12 @@ minor_behavior_changes:
 - area: golang
   change: |
     Change ``RegisterHttpFilterConfigFactoryAndParser`` to ``RegisterHttpFilterFactoryAndConfigParser``.
+- area: overload
```

Comment:
I think this section (behavior change) can be removed / merged with new features  as this is not on by default